tests/test_core/test_to_dict.py

```diff
@@ -53,6 +54,12 @@ def ensemble_samples():
         ChangePointsTrendTransform(
             in_column="target", change_point_model=Binseg(), detrend_model=LinearRegression(), n_bkps=50
         ),
+        pytest.param(
```

Comment:
Don't we want to make fail strict? Like `marks=pytest.mark.xfail(strict=True)`?

Reply:
Don't think so. In case if it  works magically that would be great)